File: src/com/eagle/programmar/Bash/Commands/Bash_PythonOption.java
// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Sep 8, 2022

package com.eagle.programmar.Bash.Commands;

import com.eagle.programmar.Bash.Symbols.Bash_Identifier_Reference;
import com.eagle.programmar.Bash.Terminals.Bash_Keyword;
import com.eagle.tokens.TokenChooser;
import com.eagle.tokens.TokenSequence;

public class Bash_PythonOption extends TokenChooser
{
	public @CHOICE static class Bash_PythonOptionM extends TokenSequence
	{
		public @S(10) Bash_Keyword M = new Bash_Keyword("-m");
		public @S(20) Bash_Identifier_Reference moduleName;
	}
}

File: csrc/com/eagle/programmar/Bash/Commands/Bash_PythonOption.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Sep 8, 2022

namespace com.eagle.programmar.Bash.Commands
{
	using Bash_Identifier_Reference = com.eagle.programmar.Bash.Symbols.Bash_Identifier_Reference;
	using Bash_Keyword = com.eagle.programmar.Bash.Terminals.Bash_Keyword;
	using TokenChooser = com.eagle.tokens.TokenChooser;
	using TokenSequence = com.eagle.tokens.TokenSequence;

	public class Bash_PythonOption : TokenChooser
	{
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @CHOICE static class Bash_PythonOptionM extends com.eagle.tokens.TokenSequence
		public class Bash_PythonOptionM : TokenSequence
		{
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(10) com.eagle.programmar.Bash.Terminals.Bash_Keyword XXopt = new com.eagle.programmar.Bash.Terminals.Bash_Keyword("-m");
			public Bash_Keyword XXopt = new Bash_Keyword("-m");
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(20) com.eagle.programmar.Bash.Symbols.Bash_Identifier_Reference XXmoduleName;
			public Bash_Identifier_Reference XXmoduleName;
		}
	}

}
